poster/index.html

```diff
@@ -1,114 +1,108 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
 
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>poster</title>
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100|Megrim|Raleway:200,900" rel="stylesheet">
     <link rel="stylesheet" href="styles/styles.css">
-    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:100" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Megrim" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Raleway:900" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Raleway:200" rel="stylesheet">
 </head>
 
 <body>
     <div class="outerwrapper">
-        <div class="tophalf">
-            <h1>
-                <div class="titlefirstletter">B</div>
-                <div class="titlesecondletter">A</div>
-                <div class="titlethirdletter">L</div>
-                <div class="titlefourthletter">L</div>
-                <div class="titlefifthletter">e</div>
-                <div class="titlesixthletter">T</div>
-                <div class="titleseventhletter">T</div>
-            </h1>
-        </div>
+        <h1 class="tophalf">
```

Comment:
It's possible to combine your `tophalf` class with your `h1` element, which simplifies your markup.